es-design-system/pages/molecules/es-popover.vue

```diff
@@ -21,16 +21,22 @@
                 </b-link>
                 for more details.
             </p>
-            <div class="my-450">
```

Comment:
We generally do have `my-450` around each category of example(s) on a page to give it room and to visually separate the examples - I'd recommend keeping that around each variant here.